openslides/motions/static/templates/motions/motion-list.html

```diff
@@ -45,21 +45,27 @@ <h1 translate>Motions</h1>
           <span class="caret"></span>
         </button>
         <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownExport">
-          <li>
+          <li os-perms="motions.can_manage">
```

Comment:
This is wrong. PDF Export should be possible for all users with motions.can_see permissions.